_publications/shkurti2019cocomd.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "CoCo-MD: A Simple and Effective Method for the Enhanced Sampling of Conformational Space"
+collection: publications
+permalink: /publications/shkurti2019cocomd
+type: pub
+date: 2019-01-08
+author: "Ardita Shkurti, Ioanna Danai Styliari, Vivek Balasubramanian, Iain Bethune, Conrado Pedebos, Shantenu Jha, and Charles A. Laughton"
+venue: "Journal of Chemical Theory and Computation, 2019"
+paperurl: https://pubs.acs.org/doi/pdf/10.1021/acs.jctc.8b00657
```

Comment:
The link does not point to the publication site, but the pdf. It should point to where the paper is published.

Reply:
Done. Thanks for checking.